guides/source/webpacker.md

```diff
@@ -51,7 +51,7 @@ To use Webpacker, you must install the Yarn package manager, version 1.x or up,
 
 NOTE: Webpacker depends on NPM and Yarn. NPM, the Node package manager registry, is the primary repository for publishing and downloading open-source JavaScript projects, both for Node.js and browser runtimes. It is analogous to rubygems.org for Ruby gems. Yarn is a command-line utility that enables the installation and management of JavaScript dependencies, much like Bundler does for Ruby.
 
-Webpacker is installed by default in Rails 6.0 and up. You can install it with a new project in some older versions by adding `--webpack` to the `rails new` command. Webpacker can be added in an existing project by running `bin/rails webpacker:install`. This installation command creates the following local files:
+Webpacker is installed with a new project by adding `--webpack` to the `rails new` command. Webpacker can be added in an existing project by running `bin/rails webpacker:install`. This installation command creates the following local files:
```

Comment:
Since the [`webpacker:install`](https://github.com/rails/webpacker/blob/v6.0.0.rc.5/lib/tasks/webpacker/install.rake) task is not part of Rails by default, I wanted to clarify these instructions.  What do you think about the following rewording?

```suggestion
To include Webpacker in a new project, add `--webpack` to the `rails new` command. To add Webpacker to an existing project, add the `webpacker` gem to the project's `Gemfile`, run `bundle install`, and then run `bin/rails webpacker:install`.

Installing Webpacker creates the following local files:
```